prac_06/guitar.py

```diff
@@ -0,0 +1,32 @@
+"""
+CP1404 Practical
+
+Estimate: 5 minutes
+Actual:   10 minutes
+
+"""
+
+CURRENT_YEAR = 2024
```

Comment:
I know from the instructions we were told to hardcode the year into the code, however one minor improvement would be to provide a dyamic calculation of the current year